pvlib/pvsystem.py

```diff
@@ -1580,15 +1580,21 @@ def sapm(effective_irradiance, temp_cell, module):
     Parameters
     ----------
     effective_irradiance : numeric
-        Effective irradiance (suns).
+        Effective irradiance [W/m^2].
 
     temp_cell : numeric
-        The cell temperature (degrees C).
+        Cell temperature [C].
 
     module : dict-like
         A dict or Series defining the SAPM parameters. See the notes section
         for more details.
 
+    irrad_ref : float (optional, default=1000)
+        Reference irradiance [W/m^2].
+
+    temp_ref : float (optional, default=25)
```

Comment:
Will people find this kwarg useful for this model? I'd rather not expose it unless there's a demand.

Reply:
Similar to above, I'd prefer something like `reference_temperature`.

Reply:
`irrad_ref` and `temp_ref` are the current kwarg names for the same quantities in the Desoto, CEC and pvsyst functions. I'm OK changing them but let's do all together and not in this PR.

I do think that exposing the `temp_ref` quantity is useful, in a minority of cases. Some of the parameter sets in the old SAPM database are referenced to 50C rather than 25C.